Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'json'
+gem 'ruby-beautify'
```

Comment:
What ever is easier I tried to make it work without it but the generated templates looked a mess